src/lib/krb5/os/os-proto.h

```diff
@@ -58,6 +58,7 @@ struct server_entry {
     k5_transport transport;     /* May be 0 for UDP/TCP if hostname set */
     char *uri_path;             /* Used only if transport is HTTPS */
     int family;                 /* May be 0 (aka AF_UNSPEC) if hostname set */
+    int master;                 /* Set to true/false by URI RR. -1 when unset. */
```

Comment:
This line runs over 79 characters.  Also, there may be other information sources in the future which can set the flag, so the comment shouldn't talk about URIs.  I suggest "True, false, or -1 for unknown."

Similarly, the commit message should mention URI lookups only when talking about the motivation for the change.  After that, it should say "... not been determined by the information source" and "If the flag is set, use it...".